accessibility.md

```diff
@@ -31,7 +31,7 @@ This product uses standard Windows navigation keys.
 ## Accessibility features in the product documentation
 {: #doc-accessibility}
 
-Accessibility features help people with a physical disability, such as restricted mobility or limited vision, or with other special needs, use information technology products successfully.
+Accessibility features help people with a physical disability, such as restricted mobility or limited vision, or with other accommodations, use information technology products successfully.
```

Comment:
@bellabie 
Thanks for the PR. But "accomodations" doesn't work because accommodations aren't something that people _have_, they're something people _need_.

I suggest removing that phrase altogether. What do you think?
>Accessibility features assist users who have a disability, such as restricted mobility or limited vision, to use information technology content successfully.

Reply:
Good point! You could say "accommodation needs," perhaps, if there's a reason preserving it proves necessary.

Reply:
I made the change in https://github.com/IBM-Bluemix-Docs/watson-knowledge-studio/pull/12